ccmlib/dse_node.py

```diff
@@ -104,7 +104,7 @@ def start(self,
         if profile_options is not None:
             config = common.get_config()
             if not 'yourkit_agent' in config:
-                raise NodeError("Cannot enable profile. You need to set 'yourkit_agent' to the path of your agent in a ~/.ccm/config")
+                raise NodeError("Cannot enable profile. You need to set 'yourkit_agent' to the path of your agent in a %s/config" % common.get_default_path_display_name())
```

Comment:
This method of string formatting isn't Python-3 compliant. Can you use `.format()` instead?